app/views/virtual_workshop_invitation_mailer/attending.html.haml

```diff
@@ -15,14 +15,17 @@
                 %h3 Hi #{@member.name},
                 %p.lead
                   - if @waiting_list
-                    A spot became available and your attendance has now been confirmed.
+                    A spot became available and your attendance has now been confirmed!
                   - else
-                    Your spot has been confirmed.
+                    Your spot has been confirmed!
                     - if @invitation.role.eql?('Student')
-                      %p If you haven't already, please remember to let us know what you will be working on through the invitation. Projects that involve specific frameworks, libraries, or languages that are not part of the codebar tutorials (anything other than Ruby or Javascript) are welcome, but we cannot guarantee that a coach will be available to help you. We recommend having a backup option to work on, such as codebar tutorials or katas (codewars.com or exercism.io). See you at our workshop!
+                      %p If you haven't already, please let us know what you will be working on through the invitation. Projects that involve specific frameworks, libraries, or languages that are not part of the codebar tutorials (anything other than Ruby or Javascript) are welcome, but we cannot guarantee that a coach will be available to help you. We recommend having a backup option to work on, such as codebar tutorials or katas (codewars.com or exercism.io).
                   %p
                     If you can no longer make it, please cancel your invitation <strong>before 15:00</strong> on the day of the workshop.
-                    <a href='https://codebar.io/student-guide#attendance'>We have a three-strikes attendance policy for no-shows</a> but due to high demand for student spots you might receive a notification if you miss any Virtual workshops.
+                    <a href='https://codebar.io/student-guide#attendance'>We have a three-strikes attendance policy for no-shows</a> but due to high demand for student spots, you might receive a notification if you miss any virtual workshops.
```

Comment:
@KimberleyCook it's not clear to me what this sentence means, what is a notification in this context?

Reply:
I think it means email?

Reply:
Ok but you would get a strike notification or not so I'm not sure how this is relevant here or what it's trying to say. Maybe remove that part and just keep the <a href='https://codebar.io/student-guide#attendance'>We have a three-strikes attendance policy for no-shows</a> bit like in the other email?